provider/resource.go

```diff
@@ -253,21 +254,25 @@ func getLogGroupNamesHttpApisHelper(conn AwsApiGatewayProvider, apiStageMappingV
 func verifyAccessLogFormat(format string, apiIdWithStageName string, logGroupName string,
 	accessLogFormatKeysMap map[string]AccessLogFormatMap, mapDiagnostics *MapDiagnostics) bool {
 
-	var parsed map[string]string
+	re := regexp.MustCompile(`:\s*(\$context[.\w]*)`)
```

Comment:
a unit test with some examples will be good.

Reply:
added

Reply:
added